spec/integration/aws_elasticsearch_domain_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'spec_helper'
+
+def policy(name)
+  <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Deny",
```

Comment:
You should be able to get `user_id` with `user_id = iam_client.get_user.user.user_id`